website/docs/proof-system/proof-system.md

```diff
@@ -5,68 +5,71 @@ slug: ./
 
 # Proof System Overview
 
+<center> <img src="assets/receipt.png" width="600"/> </center>
+<br/>
 When the RISC Zero [zkVM] executes, it produces a [Receipt] that serves as a proof of validity of a given [Session].
 
-To confirm that a [Receipt] was honestly generated, use [Receipt::verify] and supply the [ImageID] of the code that should have been executed as a parameter.
+To confirm that a [Receipt] was honestly generated, use [Receipt::verify()].
 
 The [Receipt] includes a [journal], which contains the public outputs of the [Session].
 The contents of the [journal] are specified by calling [env::commit()] and [env::commit_slice()] in the [guest] code.
 
 In addition to the journal, a receipt makes a number of claims about the program execution.
 These claims are summarized in the [ReceiptClaim].
 
-A [Receipt] can take two main forms:
+### Types of Receipts
+A [Receipt] can take four main forms.
 
-- It can be represented by a collection of [SegmentReceipts], each of which proves a single [Segment].
-  Collectively, the [SegmentReceipts] prove the validity of the full [Session].
-- It can also be represented by a single [SuccinctReceipt], proving the validity of the entire session.
-  Using recursive proving, any number of [SegmentReceipts] can be compressed into a single [SuccinctReceipt].
+1. A **[flat receipt]** is a vector of [ZK-STARK]s, one for each [segment]. Segments & segment proofs are constructed by the [RISC-V Circuit].
```

Comment:
We renamed this to "Composite" receipt

Reply:
i.e. for 0.20?

Reply:
Checking with @nategraf but I thought we did that in `0.19`

Reply:
I wrote this based on what's written [here](https://docs.rs/risc0-zkvm/0.19.1/risc0_zkvm/enum.InnerReceipt.html)

Reply:
OK, so the `docs/` directory is considered (unversioned), so I'm not sure how we should deal with name changes like this.

Reply:
I'm not strongly opinionated, but I guess it seems nice to me to maintain versioned history for these proof system docs.